dev/io.openliberty.faces.internal/bnd.bnd

```diff
@@ -16,6 +16,8 @@
 
 bVersion=1.0
 
+Implementation-Version: 2.3.10
```

Comment:
This doesn't look to be used?

Reply:
Ah, interesting. 

This change was from your commit: https://github.com/pnicolucci/open-liberty/commit/a8c0e79cc18096868a70f4523b605144951e6a94#diff-fe0c218a2f00808c842c8190a1f5dd400113f8a5e925ff282c4678c7c81c3d99R17

But this bnd did use `Implementation-Version` previously.  Now we have `com.ibm.ws.org.apache.myfaces.2.3;version=latest,\` instead.

Reply:
Done under: https://github.com/OpenLiberty/open-liberty/commit/4b6d24b3c22518215bb431376f1668847d113b36

Reply:
I'll remove it.